File: server/model/system/request/attachment.go
package request

// AttachmentDeletePayload 是附件批量删除入参。
// RemoveSource 预留：是否删除真实文件，当前后端不处理。
type AttachmentDeletePayload struct {
	IDs          []uint `json:"ids"`
	RemoveSource bool   `json:"removeSource"`
}

File: web/src/api/system/attachment.ts
import request from "@/utils/request";

export const attachmentListApi = (params?: any) => request.get("/system/attachment/index", { params });
// 表格选中的 rowKey 可能是 string，后端 ids 是 uint 数组，这里统一转数字。
export const attachmentDeleteApi = (data: { ids: Array<number | string>; removeSource?: boolean }) =>
  request.post("/system/attachment/delete", { ...data, ids: data.ids.map(Number) });
